### DISCUSSION TITLE
ZWave.Me Z-Station?

### DISCUSSION THREAD

**Discussion by Botched1** [zwave-js/zwave-js-ui#3679] [Q&A]
Does anyone use one of these new Z-Stations with ZWave JS?
https://z-wave.me/products/z-station/

If so, any comments on what the right values are for:
"Normal Power Level"
"Measured output power at 0 dBm"

Thanks!

**Comment by kpine** [comment]
You should ask the manufacturer (Z-Wave.me) for those values.

**Reply by Botched1** [reply]
I have. Waiting on a response.

I thought I would ask here in case someone else already has though.

**Reply by kpine** [reply]
I would not be surprised if you are the first to be using the Z-Station with Z-Wave JS.

**Reply by kpine** [reply]
I would be surprised if the device doesn't ship with the correct values. What are they currently set to?

**Reply by Botched1** [reply]
I saw someone comment on the home assistant forums that they migrated from an aotech zstick 5 to this.

So I might not be the only one. :-) 

But to your point, I don't think there's a lot of these floating around out there... So a bit of uncharted waters.

I'm currently using a zooz Z-Wave 800 USB controller, but I really like the idea of an external controller with proper antenna so I thought I would give this a try.

**Reply by kpine** [reply]
Ah, yeah I recall that post how. It's an interesting device for sure.

**Reply by Botched1** [reply]
I'll double check the default settings tomorrow when I'm in front of it.

**Reply by Botched1** [reply]
For the record, here is what the Z-Wave.Me Z-Station looks like in zwave-js-ui 9.11.1:

![image](https://github.com/zwave-js/zwave-js-ui/assets/19985320/6f1b798e-9305-4c6d-a89f-3e915f788c91)
